diff --git a/tools/cmake/include/BoostRoot.cmake b/tools/cmake/include/BoostRoot.cmake
index e93f9071..d07401ed 100644
--- a/tools/cmake/include/BoostRoot.cmake
+++ b/tools/cmake/include/BoostRoot.cmake
@@ -261,6 +261,8 @@ while(__boost_libs_to_scan)
 
 endwhile()
 
+set(__boost_include_libraries ${__boost_include_libraries} PARENT_SCOPE)
+
 # Installing targets created in other directories requires CMake 3.13
 if(CMAKE_VERSION VERSION_LESS 3.13)
 
